launcher/build.gradle

```diff
@@ -1,11 +1,20 @@
 import com.airbnb.mvrx.JacocoReportTask
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-kapt'
 apply plugin: "jacoco"
 apply plugin: "com.vanniktech.maven.publish"
 
+tasks.withType(KotlinCompile).all {
```

Comment:
I think this can be done in the `AndroidMetaModulePlugin` (which is applied to all projects already) to avoid duplication

Reply:
@elihart We don't want this applied to any of the other sample modules.